src/Shout/SHRBTextStyler.class.st

```diff
@@ -1076,8 +1076,15 @@ SHRBTextStyler >> visitParseErrorNode: anErrorNode [
 
 { #category : #visiting }
 SHRBTextStyler >> visitPragmaNode: aPragmaNode [
+	"Lets imagin this pragma: `<  test: #toto test2: 2 >`
 
-	aPragmaNode selectorParts with: aPragmaNode keywordsPositions do: [ :keyword :position | self addStyle: #pragma from: position to: position + keyword size ].
+	We want to highlight the `<`, the `test:`, the `test2:` and the `>` but we do not want to highlight the arguments."
+
+	self addStyle: #pragma from: aPragmaNode left to: aPragmaNode left.
+	aPragmaNode selectorParts
```

Comment:
Maybe something like this could simplify more (untested)

`aPragmaNode keywordsIntervals do: [ : i | self addStyle: #pragma from: i start to: i stop ]`